spec/unit/mutant/reporter/null_spec.rb

```diff
@@ -2,21 +2,11 @@
   let(:object) { described_class.new     }
   let(:value)  { instance_double(Object) }
 
-  describe '#report' do
-    subject { object.report(value) }
+  %i[progress report start warn].each do |name|
+    describe "##{name}" do
+      subject { object.public_send(name, value) }
 
-    it_should_behave_like 'a command method'
-  end
-
-  describe '#warn' do
-    subject { object.warn(value) }
-
-    it_should_behave_like 'a command method'
-  end
-
-  describe '#progress' do
-    subject { object.progress(value) }
-
-    it_should_behave_like 'a command method'
+      it_should_behave_like 'a command method'
+    end
```

Comment:
This feels too DRY to me. I think I prefer the previous form where the same form was duplicated for each described method. Part of the reason I feel strongly about this being *too* DRY is that you're using a shared context inherited from devtools. After I parsed what the heck was going on here I felt silly because the tests are making fairly trivial assertions.